lpc55_sign_bin/src/main.rs

```diff
@@ -199,33 +199,66 @@ enum Command {
         #[clap(long, default_value_t = 0)]
         beacon: u16,
     },
+    /// Generates a TOML file encapsulating certificate settings, which can then
+    /// be passed to the --cert-cfg option for other subcommands.
+    GenCertCfg {
+        /// Path to private key, if available.
+        #[clap(long)]
+        private_key: Option<PathBuf>,
+        /// Path to root cert; can be repeated.
+        #[clap(long)]
+        root_cert: Vec<PathBuf>,
+        /// Path to signing cert; can be repeated.
+        #[clap(long)]
+        signing_cert: Vec<PathBuf>,
+    },
 }
 
 #[derive(Debug, Parser)]
 struct CertArgs {
-    #[clap(long, requires = "root_cert")]
+    /// Path to private key file, if available. Not all operations require this.
+    ///
+    /// Cannot be combined with the --cert-cfg option.
+    #[clap(long)]
     private_key: Option<PathBuf>,
-    #[clap(long, requires = "private_key")]
-    root_cert: Option<PathBuf>,
-
+    /// Path to root certificate. Can be repeated up to 4 times.
+    ///
+    /// This interface will also reuse the root certificates as the signing
```

Comment:
This doc is ambiguous to me – if we pass > 1 root certificate, which one is used for signing?

Reply:
No clue! The first one I guess? This behavior was already present in the TOML file interface and was not documented there.

Reply:
I figured it out, see [this comment](https://github.com/oxidecomputer/lpc55_support/pull/68/files#r1199301158) about an issue here

(basically, `root_cert` and `signing_cert` are only the same if there's **only one** certificate)